src/app/contact/contact.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="contact-us">
+  <div class="header">
+    <h3>KONTAKT</h3>
+    <p>Skontaktuj się z nami. Wyślij do nas wiadomość</p>
+  </div>
+  <div class="container_1">
+    <div class="contact_info">
+      <div class="row">
+        <p><i class="fa fa-map-marker fa-2x"></i> Adres: Warszawa, PL</p>
+      </div>
+      <div class="row">
+        <p><i class="fa fa-phone fa-2x"></i> Telefon: +48 997</p>
+      </div>
+      <div class="row">
+        <p><i class="fa fa-envelope fa-2x"></i> Email: skskskskAardiumpl</p>
```

Comment:
te dane kontaktowe powinny zostać przeniesione do własnego komponentu, dzięki czemu będzie można użyć tego komponentu wszędzie gdzie będą potrzebne, zamiast pisać zawsze od nowa. No i też jeśli trzeba będzie coś zmienić to zmieniamy jeden plik i będzie zmienione w całej apce.